anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -646,3 +653,46 @@ def transform(self, frames, show_progress=False, normalize=True):
             return normalized_features
         else:
             return features
+
+    def power_spectrum(self, ell_frames, AniSOAP_HYPERS=None):
+        mycg = ClebschGordanReal(AniSOAP_HYPERS["max_angular"])
+        calculator = EllipsoidalDensityProjection(**AniSOAP_HYPERS)
+        if ell_frames[0].arrays is None:
```

Comment:
Why not just call it `frames`?

Reply:
`ell_frames` may better indicate that the frames needs to contain ellipsoidal parameters (quaternions and dimensions).

Reply:
Upon conversation with you, we can change it back to frames.